nixos/modules/services/misc/gitea.nix

```diff
@@ -206,6 +206,18 @@ in
         description = "HTTP listen port.";
       };
 
+      enableSocket = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Configure Gitea to listen on a unix socket instead of the default TCP port.";
+      };
+
+      permSocket = mkOption {
```

Comment:
What is your use case for permissions other than `666`? I'm not sure this actually needs an option. **If** it isn't common to change the value the sysadmin can add `services.gitea.settings.UNIX_SOCKET_PERMISSION = "666";`.

Reply:
I am using variant with `permSocket = 660;`
This variant requires manually adding permission for the nginx server.
```
  users.users.nginx = {
    extraGroups = [ "gitea"];
  };
```

Reply:
I still think this isn't necessary and `services.gitea.settings.UNIX_SOCKET_PERMISSION = "666";` is sufficient to cover this.

Reply:
I think it's better to add this protection option.